assets/src/edit-story/components/tooltip/index.js

```diff
@@ -49,8 +49,7 @@ const Tooltip = styled.div`
   justify-content: center;
   align-items: center;
   flex-direction: row;
-  white-space: nowrap;
-  will-change: transform;
+  max-width: 180px;
```

Comment:
We can change this to a different `px` breakpoint at which text wraps, or we can use `em` if we'd like to use a max amount of characters here instead

Reply:
Looking at this, I feel the multi-line text ought to be centered here and maybe have a smaller line-height - the gap seems a bit too big to me. I don't know if there's a design for this somewhere?

![Screen Shot 2020-08-26 at 14 08 15](https://user-images.githubusercontent.com/637548/91340510-2506a000-e7a6-11ea-8f00-c640bfee9b54.png)

Reply:
Yea, I'd agree with that. happy to make any and all design updates now that we see it wrapping. I wasn't looking at any when I was updating this, just getting the functionality to work

Reply:
new update:
<img width="292" alt="Screen Shot 2020-08-26 at 1 31 07 PM" src="https://user-images.githubusercontent.com/35983235/91349321-5f6e3e00-e7a2-11ea-9b03-270eb6ca402a.png">

Reply:
[Design for the line-height](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Stories-Stable?node-id=736%3A1178) reference (the color has changed meanwhile for the new design for this specific tooltip apparently but might still be good reference for the line height):
<img width="250" alt="Screenshot 2020-08-27 at 13 23 06" src="https://user-images.githubusercontent.com/3294597/91431048-17adec00-e860-11ea-92fe-18c2955aff79.png">

There's also [the general tooltip style in Figma](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=549%3A15890).

Both one- and single-line seem to have 20px as line-height.